components/KiboShippingOptions.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="sf-shipping-options">
+    <slot name="shipping-methods">
+      <SfRadio
+        v-for="(shippingMethod, index) in shippingMethods"
+        :key="`${shippingMethod.name}_${index}`"
+        :name="shippingMethod.name"
+        :value="shippingMethod.value"
+        :label="shippingMethod.label"
+        :details="shippingMethod.details"
+        :required="shippingMethod.required"
+        class="sf-radio"
+        @change="selectShipping($event)"
```

Comment:
event change should pass the shippingMethod object

Reply:
passed the shipping object